### DISCUSSION TITLE
How can I limit the canvas size

### DISCUSSION THREAD

**Discussion by jack361** [jgraph/drawio#4303] [General]
* [√ ] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ √] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

How can I limit the canvas size instead of letting the canvas expand infinitely, such as setting the canvas size to 1920×1080, where the pattern cannot be placed or the excess will not be displayed after publication

**Comment by Y121** [comment]
It works. I need it, too

**Comment by happy08** [comment]
count on more +1 for this feature

**Comment by alderg** [comment]
Please describe the requirements in more detail (eg. per file, globally, via API call, in embed mode etc).
